syz-manager/rpc.go

```diff
@@ -302,6 +314,23 @@ func (serv *RPCServer) Poll(a *rpctype.PollArgs, r *rpctype.PollRes) error {
 			f.inputs = nil
 		}
 	}
+	// Update server's reward table for MAB seed selection.
+	for sig, reward := range a.MABSSRewardDiff {
+		if _, ok := serv.mabSSRewards[sig]; !ok {
+			serv.mabSSRewards[sig] = 0.0
+		}
+		serv.mabSSRewards[sig] += reward
+		log.Logf(4, "MAB SS reward update %v: %v", sig.String(), reward)
+	}
+	// TODO(Daimeng): Send server-side reward update to fuzzer, when there are
```

Comment:
The case of multiple fuzzers is very important. In most real uses there are multiple fuzzers, sometimes I use up to 40. What will it take to resolve this TODO?